perun-core/src/test/java/cz/metacentrum/perun/core/api/AuthzResolverIntegrationTest.java

```diff
@@ -64,6 +64,378 @@ public void authorizedPerunAdmin() throws Exception {
 		assertTrue(AuthzResolver.authorized(session, "default_policy", Collections.emptyList()));
 	}
 
+	@Test
+	public void authorizedVoAdmin() throws Exception {
+		final Vo createdVo = perun.getVosManager().createVo(sess, new Vo(0,"test123test123","test123test123"));
+		final Member createdMember = createSomeMember(createdVo);
+		final User createdUser = perun.getUsersManagerBl().getUserByMember(sess, createdMember);
+
+		AuthzResolver.setRole(sess, createdUser, createdVo, Role.VOADMIN);
+
+		PerunSession session = getHisSession(createdMember);
+		AuthzResolver.refreshAuthz(session);
+		assertTrue(AuthzResolver.authorized(session, "test_authorized_vo_admin",  Arrays.asList(createdVo)));
+	}
+
+	@Test
+	public void authorizedVoAdminCycleAdmin() throws Exception {
```

Comment:
Please change the names of the tests which have aasertFalse() so they start with unauthorized prefix. It will be easier to understand what is the test checking.